src/index.ts

```diff
@@ -16,57 +20,79 @@ async function getFiles(dir: string): Promise<string[]> {
       return dirent.isDirectory() ? getFiles(res) : res;
     })
   );
+
   return Array.prototype
     .concat(...files)
     .filter((file) => path.extname(file) === ".svg");
 }
 
+const parseToJSXComponent = (
+  filePath: string,
+  outputPath: string,
+  callback?: () => any
+) => {
+  const spinner = ora("generating components").start();
```

Comment:
this will cause multiple spinner, this should be called inside command body function and updated every iteration in iterate function

Reply:
Updated, check again pak @adeyahya